core/src/main/java/uk/ac/bbsrc/tgac/miso/core/util/PaginationFilter.java

```diff
@@ -332,6 +342,16 @@ public static PaginationFilter sequencer(long id) {
     };
   }
 
+  public static PaginationFilter sequencingParameters(long parametersId) {
+    return new PaginationFilter() {
+
+      @Override
+      public <T> void apply(PaginationFilterSink<T> sink, T item, Consumer<String> errorHandler) {
+        sink.restrictPaginationBySequencingParametersId(item, parametersId, errorHandler);
+      }
+    };
+  }
+
   public static PaginationFilter user(String loginName, boolean creator) {
     return new PaginationFilter() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make this anonymous inner class a lambda [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1604)